ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,75 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return num1 == num2;
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            double result = (minuend - subtrahend);
+            return result;
         }
 
         public int Add(int number1, int number2)
         {
-            throw new NotImplementedException();
+            return (number1 + number2);
         }
 
         public int GetSmallestNumber(int number1, int number2)
         {
-            throw new NotImplementedException();
+
+            if (number1 < number2)
+            {
+                return number1;
+            }
+
+            else
+            {
+                return number2;
+            }
+
         }
 
         public long Multiply(long factor1, long factor2)
         {
-            throw new NotImplementedException();
+            return factor1 * factor2;
+
         }
 
         public string GetGreeting(string nameOfPerson)
         {
-            throw new NotImplementedException();
+            if (nameOfPerson != "")
```

Comment:
There's a number of empty lines in this method.  See about cleaning up the formatting a little to make it more standard